docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -127,14 +136,6 @@ async function renderApiDocumentation(inputDir, outputDir, uriRootDir, apiVersio
 			let filePath = path.join(outputDir, `${document.documentPath}.md`);
 
 			try {
-				// Hugo uses a special file-naming syntax to represent documents with "child" documents in the same directory.
```

Comment:
The model file was the only "index.md" file, so now that we're not rendering it at all, this handling can be removed.